docs/SRS-Volere/SRS.tex

```diff
@@ -142,15 +142,29 @@ \subsection{Style Requirements}
 
 \section{Usability and Humanity Requirements}
 \subsection{Ease of Use Requirements}
-\lips
+\begin{itemize}
+  \item \textbf{EUR1}: The app should save states between use in order to minimize user input.
+  \item \textbf{EUR2}: The app should allow users to undo changes made.
+\end{itemize}
 \subsection{Personalization and Internationalization Requirements}
-\lips
+\begin{itemize}
+  \item \textbf{PIR1}: For this version of the app, English will be the only supported language.
```

Comment:
why not just say : English will be supported ... do the requirements need to limit to only english?